docs/serving/samples/hello-world/helloworld-java-spark/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use the official maven/Java 8 image to create a build artifact.
+# https://hub.docker.com/_/maven
+FROM maven:3.5-jdk-8-alpine as builder
+
+# Copy local code to the container image.
+WORKDIR /app
+COPY pom.xml .
+COPY src ./src
+
+# Build a release artifact.
+RUN mvn package -DskipTests
+
+# Use the Official OpenJDK image for a lean production stage of our multi-stage build.
+# https://hub.docker.com/_/openjdk
+# https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+FROM openjdk:8-jre-alpine
+
+# Copy the jar to the production image from the builder stage.
+COPY --from=builder /app/target/helloworld-0.0.1-SNAPSHOT-jar-with-dependencies.jar /helloworld.jar
+
+# Service must listen to $PORT environment variable.
+# This default value facilitates local development.
+ENV PORT 8080
```

Comment:
ENV statement should be dropped along with it's comments.

Reply:
@grayside In Java example, I am reading on PORT 8080, which is not a default port. That's why I set this here in the Dockerfile.  There was a comment in the #954 regarding running it on  8080. If that's not needed, I will remove it and corresponding code? Let me know.

Reply:
right, if we want it to respect PORT, we need to define a default somewhere. this can be overridden.

Reply:
This `ENV PORT 8080` statement was previously used to set a default to facilitate local `docker run`'s. However, we found that developers misinterpreted this as documenting that their app should listen on 8080.

The formula now should be:

* Recommend developers set the value in a local docker run
* Hardcode a default in the app for 8080... or whatever you want.

Removing this PORT value has a secondary benefit in more quickly surfacing when the application is not listening to PORT at runtime, as opposed to PORT as set at container build.

Reply:
can we go ahead and take this pr and i can do a sweep on this secondary?
in different frameworks, the default is always 8080, and it's hard to force them to fail even w/o the env var.

Reply:
Follow up PR: #1204